api/overview/azure/service-bus.md

```diff
@@ -25,10 +29,24 @@ Install the [NuGet package](https://www.nuget.org/packages/WindowsAzure.ServiceB
 Install-Package WindowsAzure.ServiceBus
 ```
 
+#### .NET Core CLI
+
 ```bash
 dotnet add package WindowsAzure.ServiceBus
 ```
 
+### Code Example
+
+This example sends a message to a Service Bus queue.
+
+```csharp
+// using Microsoft.ServiceBus.Messaging;
+
+QueueClient client = QueueClient.CreateFromConnectionString(connectionString, queueName);

```

Comment:
It doesn't implement `IDisposable`.  I guess it's a factory method so they can manage the lifetime on our behalf.